modules/microshift-auto-apply-manifests.adoc

```diff
@@ -30,33 +29,32 @@ The `kustomize` tool must be either running as a separate step in the boot proce
 This example demonstrates automatic deployment of a BusyBox container using `kustomize` manifests in the `/etc/microshift/manifests` directory.
 
 .Procedure
-. Create the BusyBox manifest files by running the following commands:
+. Create the BusyBox manifest files by running the following commands.
 +
-* Define the directory location.
+.. Define the directory location:
 +
 [source,terminal]
 ----
-$ MANIFEST_DIR=/etc/microshift/manifests
+$ oc apply MANIFEST_DIR=/etc/microshift/manifests
```

Comment:
```suggestion
$ MANIFEST_DIR=/etc/microshift/manifests
```

Reply:
@zshi-redhat We don't need a oc action here? What does this command do?

Reply:
@ShaunaDiaz  Right, we don't need to apply the manifests manually, it is supposed to be applied automatically by microshift build-in kustomize service upon start-up if my understanding is correct.

Reply:
This cmd assigns the directory path to `MANIFEST_DIR` environment variable which then be used in other cmds.